### PR TITLE
Update to Threema 1.2.41

### PR DESCRIPTION
**Pull request by ljrk0**
- [x] Doesn't update node (Still at node18)
- [x] No Freedesktop SDK update available
- [x] Didn't update Threema Web dep

### CODE REVIEW

**Comment by ljrk0** [comment]
Hm, unfortunately this fails with `EAI_AGAIN` when trying to access github.com as part of the packaging process, similar to https://github.com/flatpak/flatpak-builder-tools/issues/173:

```
Starting packaging
Prepare package for linux-deb consumer
Could not create package because of an error: RequestError: getaddrinfo EAI_AGAIN github.com
```

The error is caught in `tools/run-dist-electron.js` and printed there (unfortunately w/o a backtrace).

There's also this warning earlier:

```
> app:install
> cd app && npm install && cd ..

npm info using npm@9.8.1
npm info using node@v18.18.1
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'threema-consumer-web@1.2.41',
npm WARN EBADENGINE   required: { node: '18.19.1', npm: '10.2.4' },
npm WARN EBADENGINE   current: { node: 'v18.18.1', npm: '9.8.1' }
npm WARN EBADENGINE }
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm info run electron@29.0.1 postinstall node_modules/electron node install.js
npm info run electron@29.0.1 postinstall { code: 0, signal: null }

```

So maybe the node18 SDK Extension is too old?

**Comment by dbrgn** [comment]
I believe this is probably an issue with flatpak-node-generator, I suspect there are dependencies that aren't put into `generated-sources.json` by flatpak-node-generator (maybe local `file:`-dependencies), which causes an error when npm tries to fetch them from github.com at build time.

(Regarding the Node version, you should be able to switch to Node 20 without any issues.)

By the way, latest version available is 1.2.43, the current version of Threema Desktop on FlatHub complains when starting that it is too old. (That happens when the application isn't updated for 180 days.)

File: ch.threema.threema-web-desktop.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/threema-ch/threema-web-electron.git
-        commit: d0f631164c8aba34a317fec6ee1ad56de515d0ce
+        commit: 55a1cb27b39a4a0f092be9586868c9abd59f76c2
```
Comment (dbrgn) [inline ch.threema.threema-web-desktop.yml:45]: Could be replaced by `tag: 1.2.41`, right?

Comment (loleg) [reply ch.threema.threema-web-desktop.yml:45]: `1.2.46` at this point. I've tested it (via direct RPM on Fedora), and it connects fine.

**Comment by ljrk0** [comment]
bot, build

**Pull request merged**